iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/FileUploadTests.java

```diff
@@ -270,30 +260,8 @@ public static void stopProxy()
 
     private DeviceClient setUpDeviceClient(IotHubClientProtocol protocol) throws URISyntaxException, InterruptedException, IOException, IotHubException, GeneralSecurityException
     {
-        DeviceClient deviceClient;
```

Comment:
This block is one of the many instances of the same basic logic being used to create the appropriate test identity. Since it has been centralized, there is no need for it to live here anymore.